cmd/catalyst-node/balancer/balancer.go

```diff
@@ -0,0 +1,297 @@
+package balancer
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"sync"
+
+	serfclient "github.com/hashicorp/serf/client"
+	glog "github.com/magicsong/color-glog"
+)
+
+type BalancerIface interface {
+	Start() error
+	UpdateMembers(members *[]serfclient.Member) error
+	Kill()
+	GetBestNode(redirectPrefixes []string, playbackID, lat, lon, fallbackPrefix string) (string, string, error)
+	QueryMistForClosestNodeSource(playbackID, lat, lon, prefix string, source bool) (string, error)
+}
+
+type Config struct {
+	Args                     []string
+	MistUtilLoadPort         uint32
+	MistLoadBalancerTemplate string
+}
+
+type Balancer struct {
+	config   *Config
+	cmd      *exec.Cmd
+	endpoint string
+}
+
+// create a new load balancer instance
+func NewBalancer(config *Config) *Balancer {
```

Comment:
```suggestion
func NewBalancer(config *Config) *Balancer {
```

Nit: Comment not needed, does not explain anything.